src/content/artifact-management/retention-rules.mdx

```diff
@@ -85,6 +85,11 @@ This value applies per group. So, the maximum total number of packages to kept w
 ### Deletion order
 Packages are deleted based on the date pushed, starting with the oldest and being newest packages the latest to be deleted until a total of `TOTAL_PACKAGES - retention_count_limit` packages have been deleted.
 
+<Note variant="note">
+If "Limit by size" is selected, packages are deleted based on the semantic version, starting with the lowest version. For example, when deleting by size and if “Group Packages by Name” is selected, if packages `PkgA 1.1.1`, `PkgA 1.1.2`, `PkgB 0.20251223.git-12345a` and `PkgB 0.20251224.git12345b` are uploaded, `PkgA 1.1.1` and `PkgB 0.20251223.git12345a` are deleted. 
```

Comment:
Nitpick: I'd probably restructure this slightly so that it's more like if input X and Y then output is Z, breaking it up a little